installation/resources/installer-cr-cluster.yaml.tpl

```diff
@@ -34,6 +34,11 @@ spec:
    #   namespace: "knative-build"
     - name: "knative-eventing"
       namespace: "knative-eventing"
+   # - name: "knative-eventing-channel-kafka"
+   #   namespace: knative-eventing
+   #   source:
+   #     # NOTE: this is a go-getter compatible url
+   #     url: github.com/kyma-incubator/knative-kafka.git//resources/knative-kafka?ref=master
```

Comment:
is it safe to pin this to master?

Reply:
No it is not. I pinned to a git commit before, but because the referenced docker images inside the helm template are tagged with `latest`, I had a mismatch between code and template where the new docker image required an environment variable which my helm template did not provide (because was pinned to git commit). I talked to `knative-kafka` authors and asked for tagged docker images inside the helm chart. If we have them, I will pin the `ref` to a git commit instead. Until then, `master` is still better than pinning to a git commit. cc: @davyodom 

What about I create a tracking ticket to not forget about the pinning and we proceed with master ref for now?

Reply:
https://github.com/kyma-project/kyma/issues/7008